.gitignore

```diff
@@ -1,2 +1,2 @@
-buildlist.json
+*.json
```

Comment:
The new `adlr-cli` commands produce json output files so I updated this to ignore all json files not just the buildlist